tree/dataframe/inc/ROOT/RDF/ActionHelpers.hxx

```diff
@@ -201,30 +201,25 @@ public:
    }
 };
 
-template <typename ProxiedVal_t>
-class R__CLING_PTRCHECK(off) ReportHelper : public RActionImpl<ReportHelper<ProxiedVal_t>> {
+template <typename RNode_t>
+class R__CLING_PTRCHECK(off) ReportHelper : public RActionImpl<ReportHelper<RNode_t>> {
    const std::shared_ptr<RCutFlowReport> fReport;
-   // Here we have a weak pointer since we need to keep track of the validity
-   // of the proxied node. It can happen that the user does not trigger the
-   // event loop by looking into the RResultPtr and the chain goes out of scope
-   // before the Finalize method is invoked.
-   std::weak_ptr<ProxiedVal_t> fProxiedWPtr;
+   RNode_t *fNode;
```

Comment:
Maybe a comment saying that the raw pointer is enough thanks to the current ownership model would be helpful